### DISCUSSION TITLE
Possibility to change the minimum PHP requirement to 7.4

### DISCUSSION THREAD

**Discussion by chrisAlkemy** [filamentphp/filament#1690] [General]
This project is amazing and very useful; however, technology companies in Latin America, have weird software policies and do not allow to migrate to PHP 8.

Is there a possibility to give compatibility as in version 1.x?

**Comment by danharrin** [comment]
Unfortunately, I feel like supporting 7.4 would hinder the progress of our software, as we are not using the latest features available to us to keep the code clean and maintainable. PHP 7.4 is no longer fully maintained with bug fixes either. I see this as more of a problem with your software policies at work I'm afraid, but I have total empathy for you as a developer - this sounds like hell :(

**Comment by ryangjchandler** [comment]
PHP 7.4 is now in security-fix only mode and will stop receiving those security fixes in November 2022. 

For this reason, we'll be sticking to PHP 8.0+.

**Comment by chrisAlkemy** [comment]
Thank you for your answers. I think I will have to get more "creative" until November. haha
